docs/chaos/behavior.md

```diff
@@ -68,10 +68,10 @@ var pipeline = new ResiliencePipelineBuilder()
 
 ## Defaults
 
-| Property             | Default Value | Description                                    |
-|----------------------|---------------|------------------------------------------------|
-| `OnBehaviorInjected` | `null`        | Action executed when the behavior is injected. |
-| `BehaviorGenerator`  | `null`        | Custom behavior to be injected.                |
+| Property             | Default Value | Description                                                                                                                  |
+|----------------------|---------------|------------------------------------------------------------------------------------------------------------------------------|
+| `BehaviorGenerator`  | `null`        | This **mandatory** delegate allows you to inject custom behavior by utilizing information that is only available at runtime. |
```

Comment:
Personally I don't think the emphasis here is needed, and it looks "shouty". "Required" would fit better with language we've used elsewhere too?

Reply:
Done